modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -64,7 +64,7 @@
     public String defaultValue;
     public String arrayModelType;
     public boolean isAlias; // Is this effectively an alias of another simple type
-    public boolean isString, isInteger, isLong, isNumber, isNumeric, isFloat, isDouble, isDate, isDateTime, isShort, isUnboundedInteger, isBoolean;
+    public boolean isString, isInteger, isLong, isNumber, isNumeric, isFloat, isDouble, isDate, isDateTime, isShort, isUnboundedInteger, isPrimitiveType, isBoolean;
```

Comment:
Why this change? Doesn't seem to be set/used.
Also how can a model be primitive?

Reply:
I would expect this to be an auto property combining all known OAS3 primitive types, e.g.
```java
isString || isInteger || isLong || isNumber || isNumeric || isFloat || isDouble || isDate || isDateTime || isShort || isUnboundedInteger ...
```

Reply:
while this doesn't look to be part of Dart2 functionality, it makes sense to add this, as concept of `isPrimitiveType` is known in other generators like in cpp and rust. This change adds functionality of `isPrimitiveType` check to _all_ generators. Going to keep it here.

Reply:
But it is not being set anywhere, this is missleading

Reply:
I agree that this should be a separate PR and get discussed between other generator implementers